shuup/admin/utils/urls.py

```diff
@@ -221,7 +224,11 @@ def get_model_url(object, kind="detail", user=None,
     :rtype: str
     """
     for module in get_modules():
-        url = module.get_model_url(object, kind, shop)
+        try:
+            url = module.get_model_url(object, kind, shop)
+        except Exception as e:
+            LOGGER.exception(sr(e))
```

Comment:
I think you should target the correct exception here, maybe NoReverseMatch. But please check about the `raise_permission_denied` parameter from this function, maybe you should respect that as well